src/main/java/Task.java

```diff
@@ -0,0 +1,23 @@
+import java.awt.image.SinglePixelPackedSampleModel;
+
+public class Task {
+    protected String name;
+    protected boolean done;
+
+    public Task(String input){
+        this.name = input;
+        this.done = false;
+    }
+    public void taskDone(){
```

Comment:
Suggestion:
Name the method as setTaskAsDone instead of taskDone to make it sound more intuitive